### DISCUSSION TITLE
Unable to use verified domain for github pages

### DISCUSSION THREAD

**Discussion by mo-shawa** [community/community#42769] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi all, 

As the title says, I'm unable to use a verified custom domain for my portfolio deployment, the error I get is this:

The custom domain `<my domain>` is already taken. If you are the owner of this domain, check out https://docs.github.com/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages for information about how to verify and release this domain. 

I had already deployed another static site to that domain, from the same repository, which I overwrote with a force push if that is relevant at all. 

If you've run into anything like this or have any ideas please let me know!

**Comment by BernnyVelasquez** [comment]
Did you ever figure this out?
